src/material-examples/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare var module: {id: string};
```

Comment:
The moduleId should be removed by the inline resources script. Why do we need it at all?